java/integTest/src/test/java/glide/PubSubTests.java

```diff
@@ -135,21 +135,31 @@ private BaseClient createClient(boolean standalone) {
     @SneakyThrows
     public void cleanup() {
         for (var client : clients) {
-            if (client instanceof GlideClusterClient) {
-                ((GlideClusterClient) client)
-                        .customCommand(new GlideString[] {gs("unsubscribe")}, ALL_NODES)
-                        .get();
-                ((GlideClusterClient) client)
-                        .customCommand(new GlideString[] {gs("punsubscribe")}, ALL_NODES)
-                        .get();
-                ((GlideClusterClient) client)
-                        .customCommand(new GlideString[] {gs("sunsubscribe")}, ALL_NODES)
-                        .get();
-            } else {
-                ((GlideClient) client).customCommand(new GlideString[] {gs("unsubscribe")}).get();
-                ((GlideClient) client).customCommand(new GlideString[] {gs("punsubscribe")}).get();
+            try {
+                if (client instanceof GlideClusterClient) {
+                    String[] clusterCommands = {"unsubscribe", "punsubscribe", "sunsubscribe"};
+                    for (String command : clusterCommands) {
+                        try {
+                            ((GlideClusterClient) client)
+                                    .customCommand(new GlideString[] {gs(command)}, ALL_NODES)
+                                    .get();
+                        } catch (Exception e) {
+                        }
+                    }
+                } else {
+                    String[] standaloneCommands = {"unsubscribe", "punsubscribe"};
+                    for (String command : standaloneCommands) {
+                        try {
+                            ((GlideClient) client).customCommand(new GlideString[] {gs(command)}).get();
+                        } catch (Exception e) {
+                        }
+                    }
+                }
+            } catch (Exception e) {
+                // General catch block in case any other exception occurs during client cleanup
+                System.err.println("Error during cleanup for client: " + e.getMessage());
```

Comment:
Don't suppress an exception. Junit catches them and reports.

Reply:
I am waiting for your pr actually, I added this without noticing you added a fix
Can you also add a fix to node?

Reply:
Will do. Why it doesn't fail on python? Something it missing there.